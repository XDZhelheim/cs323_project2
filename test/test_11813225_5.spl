int rubbish()
{
  int aa;
  float bb;
  int aa;
  if (aa > bb)
  {
    return 'v';
  }
}