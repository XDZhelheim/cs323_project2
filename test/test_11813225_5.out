Error type 3 at Line 5: a variable is redefined in the same scope
Error type 7 at Line 6: unmatching operands
Error type 8 at Line 8: a functionâ€™s return value type mismatches the declared type
